### PR TITLE
Remove the topics tagged/view tables from sections

### PR DESCRIPTION
**Pull request by jonathan-best**
Remove the topics tagged/view tables from sections - commented out so that they can be later returned when data is ready

https://trello.com/c/sYcTNBg9

### CODE REVIEW

File: src/shared/handlers/SectionView.js [hunk 1]
```diff
@@ -195,6 +196,7 @@ class SectionView extends React.Component {
               </Col>
             </Row>
           </ChunkWrapper>
+          */}
```
Comment (radiodario) [inline src/shared/handlers/SectionView.js]: don't commit commented code - shouldn't we hide this behind a feature flag?

File: src/shared/components/ChunkWrapper.js [hunk 2]
```diff
@@ -11,6 +11,10 @@ export default class ChunkWrapper extends React.Component {
       borderBottom: '1px solid #ccc'
     };
 
+    if (!this.props.featureflag) {
```
Comment (DanForys) [inline src/shared/components/ChunkWrapper.js]: Would it be better to call this property `enabled`?

File: src/shared/components/ChunkWrapper.js [hunk 3]
```diff
@@ -11,6 +11,10 @@ export default class ChunkWrapper extends React.Component {
...
+      return <div></div>
```
Comment (DanForys) [inline src/shared/components/ChunkWrapper.js]: Better to return `null` rather than render a block level element?

Comment (jonathan-best) [inline src/shared/components/ChunkWrapper.js]: I dont think you can render nothing can you?

File: src/shared/components/ChunkWrapper.js (hunk 2 above)
Comment (jonathan-best) [inline src/shared/components/ChunkWrapper.js]: Can do, this just keeps it inline with how it was used in the past - i can change it if you feel strongly

Comment (DanForys) [inline src/shared/components/ChunkWrapper.js]: Seems to be the [officially approved way](https://facebook.github.io/react/blog/2014/07/17/react-v0.11.html#rendering-to-null)

File: src/shared/components/ChunkWrapper.js (hunk 3 above)
Comment (DanForys) [inline src/shared/components/ChunkWrapper.js]: Seems to be the [officially approved way](https://facebook.github.io/react/blog/2014/07/17/react-v0.11.html#rendering-to-null)

File: src/shared/components/ChunkWrapper.js (hunk 2 above)
Comment (DanForys) [inline src/shared/components/ChunkWrapper.js]: Consistency wins. That's fine then :dancer:

**Comment by StuartFT** [comment]
:+1:

**Pull request merged**